### PR TITLE
cattle 1.2.0

### PR DESCRIPTION
**Pull request by DomT4**
Version bump, added head, fixed single quotes, notes the new clang failure with upstream report link.

Afraid I failed on the test here, If someone wants to teach me some C I have no objections ;).

### CODE REVIEW

File: Library/Formula/cattle.rb [hunk 1]
```diff
@@ -1,24 +1,29 @@
...
 
-  depends_on 'glib'
-  depends_on 'gobject-introspection'
-  depends_on 'gtk-doc' => :build
-  depends_on 'pkg-config' => :build
+  # https://github.com/andreabolognani/cattle/issues/2
+  fails_with :clang do
+    build 600
+    cause "Unexpected nil-return of values"
+  end
```
Comment (mikemcquaid) [inline Library/Formula/cattle.rb]: Probably want to hold off adding this new version until this is patched upstream.

Comment (DomT4) [inline Library/Formula/cattle.rb]: Sure, happy to wait on upstream here. This new release is from last October, and as far as I can see nobody has submitted it to us till now, so I don't think there's a particularly huge hankering for the update right this second.

**Comment by DomT4** [comment]
Closed per upstream's silence on the bug report and the upstream repo being inactive since the October release. Will create a new PR once I hear something back from upstream on the Clang failures.

**Comment by bfontaine** [comment]
I just submitted a PR to upstream to fix the build, you can use the patch here in the meantime: https://github.com/andreabolognani/cattle/pull/3.diff

**Comment by DomT4** [comment]
@bfontaine Thanks Baptiste. Have reopened this and pushed your patch into the formula. Appreciated.

**Comment by DomT4** [comment]
@BrewTestBot test this please

**Comment by DomT4** [comment]
It'd fallen off the build queue again. Got lost in the breakage the other day and the bot hasn't quite looped back over everything since.

**Comment by mikemcquaid** [comment]
@BrewTestBot test this please

**Comment by xu-cheng** [comment]
Wow, this PR has been forgotten for so long time. @DomT4 you need to re-push it to rebuild the bottle.(original one has been lost in time).

**Comment by DomT4** [comment]
You can ask the bot to check it over again by calling it, but I'll bump the SHA1 to SHA256 and trigger it that way.

**Comment by bfontaine** [comment]
Another trick I use often is to rebase your branch on master.

**Comment by xu-cheng** [comment]
:ship:ed

**Pull request closed without merging**